etc/kayobe/inventory/group_vars/overcloud/cis

```diff
@@ -75,6 +75,12 @@ rhel9cis_max_log_file_size: 1024
 # `rhel9cis_bootloader_password_hash`
 rhel9cis_set_boot_pass: false
 
+# Disable account locking after inactivity period
+rhel9cis_inactivelock:
```

Comment:
@jackhodgkiss did it this way for a client deployment. Is there any difference?
```
---
# NOTICE: rule disabled otherwise rule will prevent access to accounts
# as it will expire passwords older than one year.
rhel9cis_rule_5_6_1_1: false
```

Reply:
I think that should work for a new deployment, but I wanted to restore the defaults on deployments where this is already applied. Which would you prefer?

Reply:
Makes sense to try to fix existing deployments if we can.